samples/display/lvgl/boards/native_posix.conf

```diff
@@ -4,3 +4,6 @@ CONFIG_SDL_DISPLAY_Y_RES=240
 
 CONFIG_KSCAN=y
 CONFIG_LVGL_POINTER_KSCAN=y
+CONFIG_LVGL_HOR_RES=320
```

Comment:
These changes should not be needed if https://github.com/zephyrproject-rtos/zephyr/pull/26827 gets merged (same for native_posix_64 and hsdk)

Reply:
@vanwinkeljan while it now works without these settings, the allocated memory is now higher than the needed amount. I think it is still better to specify such settings for `CONFIG_LVGL_BUFFER_ALLOC_STATIC` so that no memory is wasted.

Reply:
Memory will be not a problem on native_posix so no need to reduce the buffer on native_posix.
It could be an argument to lineup de SDL display resolution with the LVGL defaults.

Reply:
Even if memory is not a problem on posix, it is still a waste of memory for nothing (it actually doubles to be precise, +144Kb).